lib/plaid/models.rb

```diff
@@ -674,8 +727,13 @@ class TransactionLocation < BaseModel
 
       ##
       # :attr_reader:
-      # Public: The String ZIP code (or nil).
-      property :zip
+      # Public: The String postal code (or nil).
+      property :postal_code
+
+      ##
+      # :attr_reader:
+      # Public: The String country.
+      property :country
     end
```

Comment:
Considering that the address fields (`city`, `region`, `street`, `postal_code`, and `country`) will now be nullable in version `2019-02-14`, are there changes we need to make here? cc @be9 there looks to be nullability logic around `Plaid.relaxed_models`?

Reply:
country is nullable in some of the address objects stored in the `vcr_cassettes` - so I am guessing there is nothing to do.

@be9 please confirm.

Reply:
It's OK. However we need tests for various country items.